src/System.Runtime/src/System.Runtime.csproj

```diff
@@ -5,6 +5,7 @@
     <ProjectGuid>{56B9D0A9-44D3-488E-8B42-C14A6E30CAB2}</ProjectGuid>
     <AssemblyName>System.Runtime</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I have realized that this is no longer needed, I'll put it into the next commit that solves the build problem and any additional feedback.

Reply:
Removed.